pkg/proxy/accesslog/builder.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package accesslog
+
+import (
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/cilium/cilium/pkg/identity"
```

Comment:
`pkg/proxy/accesslog` is a public API, and is used by third-party software. Adding this file adds packages dependencies to those third-parties. Currently, the dependencies are intentionally relatively minimal:
```
$ grep -r github.com/cilium/cilium pkg/proxy/accesslog
pkg/proxy/accesslog/log.go:	"github.com/cilium/cilium/pkg/flowdebug"
pkg/proxy/accesslog/log.go:	"github.com/cilium/cilium/pkg/lock"
pkg/proxy/accesslog/log.go:	"github.com/cilium/cilium/pkg/logging"
pkg/proxy/accesslog/log.go:	"github.com/cilium/cilium/pkg/logging/logfields"
```

Could it be possible to move this file out of the `pkg/proxy/accesslog` package?

Reply:
Agreed, I'll move it into a subpackage

Reply:
Fixed. I've moved all of `accesslog/log.go` out so the dependencies are less than before.